src/couch_mrview/src/couch_mrview_http.erl

```diff
@@ -241,7 +241,8 @@ do_all_docs_req(Req, Db, Keys, NS) ->
         ),
         IsAdmin = is_admin(Db),
         Callback = get_view_callback(DbName, UsersDbName, IsAdmin),
-        couch_mrview:query_all_docs(Db, Args, Callback, VAcc0)
+        ValidatedArgs = couch_mrview_util:validate_all_docs_args(Db, Args),
```

Comment:
I ran `make eunit` and looked at coverage, but unfortunately this module looks pretty uncovered
```
couch_mrview_http  6%
```
And in particular this line does not get unit tested.

Reply:
Thanks! I added a set of http "local" query tests in a addition to just calling `query_all_docs` directly.